packages/components/src/tree-grid/index.js

```diff
@@ -45,18 +45,26 @@ function getRowFocusables( rowElement ) {
  * @param {WPElement} props.children      Children to be rendered.
  * @param {Function}  props.onExpandRow   Callback to fire when row is expanded.
  * @param {Function}  props.onCollapseRow Callback to fire when row is collapsed.
+ * @param {Function}  props.onChangeRow   Callback to fire when moving focus to a different row.
  * @param {Object}    ref                 A ref to the underlying DOM table element.
  */
 function TreeGrid(
-	{ children, onExpandRow = () => {}, onCollapseRow = () => {}, ...props },
+	{
+		children,
+		onExpandRow = () => {},
+		onCollapseRow = () => {},
+		onChangeRow = () => {},
```

Comment:
It'd be good to update the changelog for this change.

Reply:
Added in https://github.com/WordPress/gutenberg/pull/38314/commits/4c7e257ca34e19950089549d6d4539f1a299691a, but there's a merge conflict that I'll resolve when I rebase this PR tomorrow